import_obfuscation.go

```diff
@@ -62,6 +67,62 @@ func appendPrivateNameMap(pkg *goobj2.Package, nameMap map[string]string) error
 	return nil
 }
 
+func extractDebugObfSrc(pkgPath string, pkg *goobj2.Package) error {
+	if envGarbleDebugDir == "" {
+		return nil
+	}
+	for _, member := range pkg.ArchiveMembers {
+		if member.ArchiveHeader.Name != garbleSrcHeaderName {
+			continue
+		}
+
+		osPkgPath := filepath.FromSlash(pkgPath)
+		pkgDebugDir := filepath.Join(envGarbleDebugDir, osPkgPath)
+		if err := os.MkdirAll(pkgDebugDir, 0o755); err != nil {
```

Comment:
Move this MkdirAll outside the loop, because we'll always garble at least one package.

Also, you should probably do a RemoveAll and a Mkdir, because otherwise previous files which are now missing from the source could be left around. Same for previously obfuscated packages if GOPRIVATE changes.

Reply:
A bit lower in any case "return", this code is executed only once. It is worth rewriting the method so that only the search for the right "member" inside the loop will be performed.

Reply:
Sorry, I don't understand what you mean. Perhaps I can just review the code again when you're done changing it.